x/ibc/core/02-client/client/cli/cli.go

```diff
@@ -29,3 +29,22 @@ func GetQueryCmd() *cobra.Command {
 
 	return queryCmd
 }
+
+// NewTxCmd returns the command to create and handle IBC clients
+func NewTxCmd() *cobra.Command {
+	txCmd := &cobra.Command{
+		Use:                        types.SubModuleName,
+		Short:                      "IBC client transaction subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	txCmd.AddCommand(
+		NewCreateClientCmd(),
+		NewUpdateClientCmd(),
+		NewSubmitMisbehaviourCmd(),
```

Comment:
What about update proposal and upgrade client?

Reply:
update proposal is unnecessary, this functionality should be supported by the governance module. When testing x/upgrade I used the gov submit-proposal cmd

Reply:
proposals are included on the gov CLI afaik